### ISSUE TITLE
We need ideas for games and micro-games.

### ISSUE THREAD

**Issue by chadsansing** [MozillaFoundation/mpa-game-jam-challenge#4]
Help us come up with ideas for web-native games and micro-games about online safety and inclusion. You might share a few ideas that you type up or record, or you might make sketches or paper prototypes of games you'd like to see.

Describe and/or link to your ideas by commenting below.

**Comment by flukeout** [comment]
Hey @chadsansing can you drop in some links to the curriculum that you made around Privacy & Security issues? I think they could be a good source of inspiration.

**Comment by chadsansing** [comment]
Here is a kit with our Privacy & Security resources:

https://mozilla.github.io/curriculum-final/privacy-toolkit-v10.pdf

Also, here are a few lessons with games or game-like ideas in them:

- https://mozilla.github.io/curriculum-final/color-picker-challenge/#overview
- https://chadsansing.github.io/color-guessr/index.html 
- https://chadsansing.github.io/bridge-builder-game/index.html 
- https://mozilla.github.io/curriculum-final/encryption/really-private-messaging.html#overview
- https://mozilla.github.io/curriculum-final/encryption/werewolf-in-the-middle.html#overview

**Comment by chadsansing** [comment]
Another resource with games like those we might imagine here:

https://mozilla.github.io/curriculum-final/intermediate-web-lit-one/session08-external-css-tutorials.html#overview

**Comment by ekapros** [comment]
Hey @chadsansing ,

Is it ok if we aim for something larger than a microgame? I have started drafting a space adventure, where it is 3017 and the humans of Mars try to identify their origins. Some parties are interested in not disclosing this info, so finding and protecting info is essential.

This makes the game larger, but the fact that there is a story makes it easy to add pedagogical scaffolding that gives context to the learners to relate to. I can give the story line and ideas for puzzles within the story, which I think should be plug-and-play. However, they should go from easier to harder.

Let me know what you think. Also, this story could be appropriated as a board game for the offline challenge. Let me know if I should duplicate the question.

**Comment by chadsansing** [comment]
For sure, @ekapros! I wanted to keep the original invite bite-sized, but feel free to build on what you have! Sounds awesome.

**Comment by flukeout** [comment]
Hey @ekapros - do you have any of the design written up somewhere to share? I'd be interested in checking it out.

**Comment by ekapros** [comment]
Here's the first draft! @flukeout 

**Origins. Draft 0.1**

Mars, 3017. Jaonora just finished school in the small town of Troy, in the Gusev crater of the Ma’adim Vallis. She was bored. Most people of Troy were farmers, harvesting the bright white silica foam at the nearby Silica Valley. Today at school they were taught how these farmers where pioneers who cultivated the land and saved the Ma’adim Vallis from an environmental disaster, caused by the failed Space Exploration of the 25th Century.

Jaonora was bored. She had heard the stories again and again. However, she was not convinced: she believed that the farmers were not pioneers, but boring. At 15 years old, she didn’t want to settle in a farmers’ colony: she wanted to go on an adventure. Surely the Space Exploration era was not the dark age everyone said it to be, but an era of adventure and discovery. What a great time it must have been! That’s what she was thinking when she started going up the Home Plate of the Columbia Hills, instead of going home. That’s what Jaonora would do. Instead of going home after school, she would go up the hilltop and imagine of adventures.

Jaonora there records her thoughts on her audio-journal. [audio-journal added to inventory]

After her recording, Jaonora did something unusual. Instead of going home, like most days, she started wondering towards the valley. There, in a shaft, not unlike the silica harvesting shafts, she saw it. [click to add audio-disc to inventory].

The audio goes like this: a music-box children’s music starts to play for a few seconds. Then, a little girl starts reading numbers in a stern voice: Three, four, five, seven, one. Three, four, five, seven, one. Nine, two…

[34571, 34571, 92860, 76752, 92860, 76752, 17543, 17543]

Once the numbers stopped, the music resumed for a while. Then it stopped. The audio disc was otherwise totally empty. Jaonora was shocked. First of, she hadn’t seen an audio disc like that before. It seemed ancient: how on Mars did her audio recorder still _play_ this thing? Plus, it _was_ ancient; only the old mBámra had an accent as close to that of the girl in the recording, and mBámra was at least 120 years old. She decided to go find mBámra and talk to him. [click to go meet mBámra]

mB: How may I be of assistance, young lady?

J: This. [plays recording]

mB: Ah, now. I see you played that in your audio recorder. What permission does your audio app have?

[A. Auto saves to the web everything it plays.
B. Only plays files locally, on the machine. I think.
C. Permi… eh, what? I have no idea? I just press play…
D. What are permissions again?

onClick():
A. So, you may be found. B. You think? So, you’re not sure. If this is not true, you may be found. C. Definition. D. Definition.
]

J: OK. But what _is_ this?

mB: Spy code. These were transmitted in the old days through the ancient web system, the “radio”.

J: What does it mean? Who is it for?

mB: You need to hide. You’re not safe.

J: Why?

mB: They’ll find you. You’ve left a data trail. A data trail is [definition].

[Q: Does recording on an audio-journal leave a data trail?

A. Yes, always. B. Yes, but only if it saves the recording on the web. C. No, never. D. Can I have an audio-journal? Please?

onClick():
A. Hmmm, not exactly (data trail definition). B. That is correct! (data trail definition)  C. Hmmm, not exactly (data trail definition). D. Apologies, we do not currently ship to Planet Earth (location auto-detected from your IP). ]

J: Who will find me?

mB: You need to hide. You’re not safe.

Jaonora was delighted and furious at the same time. _Hide_? Of course not! All these years, she had been looking for an adventure, and now that one knocked on her door, she would _hide_? There was no way she was doing that!

However, she also decided to protect herself from whoever might be after her. She thought about what mBámra had said, about the data trail. What could be an effective way to protect my data, she wondered.

[A. Check Permissions. B. Protect files with a Password. C. All of the above. D. None of the above.]

[Password quiz. Why you need one explanation.
A good password is:
A. 123456 B. password C. My birthday D. (input) ]

—————————————————————— ——————————————————————
TO DO:

Jaonora is found by government agents. They found her by her IP. [IP location quiz].

They don’t know what’s going on either. They give her access to the State’s library. There Jaonora finds the code for the message:

[Code:
34571: Start message
92860: Abandon
40968: Do not Abandon
76752: Planet
17543: End message ]

She asks agents, who point back to the dark age of the Space Exploration and send her away. They’re busy.

Jaonora goes back to mBámra.

**Comment by ekapros** [comment]
I have used a real Mars location: https://planetarynames.wr.usgs.gov/Feature/3772

Radio spy-code transmissions: https://archive.org/details/ird059

**Comment by chadsansing** [comment]
I dig the narrative -

**Comment by flukeout** [comment]
Oh great, love the audio piece too.

**Comment by ekapros** [comment]
Thank you both! :)
Do you suggest I continue this as a file on my git? That should improve versioning, right? (yes, I know, git noob alert!)

**Comment by chadsansing** [comment]
Yes - then you can share a link back here or fork this repo and put your work in a new directory for a pull request. Is that half-way right, @flukeout?

**Comment by flukeout** [comment]
Yeah, I'd recommend starting a new github repo, and then drop the link in here! Let me know if you need any help with that.

**Comment by ekapros** [comment]
Awesome, thanks for the help both :) Will let you know if I get stuck.

**Comment by siddhartharao17** [comment]
Hey guys,
I finally finished drafting my idea so v1.0 is ready but I have hosted it on my gdoc.
Here's the link: http://bit.ly/2rhLF6

Let me know what you think about it :)

**Comment by chadsansing** [comment]
@siddhartharao17 - I like this kind of deduction game a lot. One way to play to better protect user days might be to use profiles or personas, like in the game @ksadorf [describes here](https://github.com/MozillaFoundation/mpa-offline-games-challenge/issues/4#issuecomment-305730089). By drafting a set of fictional profiles you can add more playability to the game right away for newcomers without asking them for data.

What do you think of that?

**Comment by flukeout** [comment]
I agree with @chadsansing here, Im not sure people could even produce their own history (or want to!) but the fictional profiles would make a lot of sense. Another idea would be to maybe have like a 'map of the internet' (something like in this [xkcd comic](https://xkcd.com/802/)) that would allow people to indicate where they spent more time and where they travelled without getting overly specific about what then were doing.

After filling something like that out, people could try to guess who the map belonged to. Nice work so far!

**Comment by ekapros** [comment]
Hi @chadsansing , @flukeout ,

I started and will be updating a repo at https://github.com/ekapros/origins

Plus, @flukeout , I will be in Vancouver for HCII 2017, 9-15 July http://2017.hci.international/
We could meet up to talk shop if you'd like :)

**Comment by ekapros** [comment]
Hi @chadsansing , @flukeout ,

I started and will be updating a repo at https://github.com/ekapros/origins

Plus, @flukeout , I will be in Vancouver for HCII 2017, 9-15 July http://2017.hci.international/
We could meet up to talk shop if you'd like :)

**Comment by siddhartharao17** [comment]
Hey @chadsansing and @flukeout !
Sorry for the late reply. I checked out both the suggestions and I had some difficulty understanding the profiles and card game and I think it serves a different intention on the same topic. I really liked the map idea though :)

Any inputs on how I could use the map idea as metadata in my idea and then use the deductions on it?  I don't know if I should do something like that.

**Comment by chadsansing** [comment]
@siddhartharao17, maybe start with a sketch of a web with the top 10-12 sites your players might like to visit online. Then they could mark the map or put game pieces on it to show which sites they visit in real life?

**Comment by siddhartharao17** [comment]
Got it! Will try it.
